optuna/cli.py

```diff
@@ -53,6 +53,10 @@ def get_parser(self, prog_name):
                             default='minimize',
                             help='Set direction of optimization to a new study. Set \'minimize\' '
                                  'for minimization and \'maximize\' for maximization.')
+        parser.add_argument('--exist-ok', default=False, action='store_true',
+                            help='If specified, the creation of the study is skipped '
+                                 'without any error even if the same name study already '
```

Comment:
This message has two if clauses, and it seems difficult to parse. Could you rephrase this sentence?

For instance, I think we can use `when` in this case as follows:
`If specified, the creation of the study is skipped without any errors when the study name is duplicated.`